common/djangoapps/enrollment/data.py

```diff
@@ -1,20 +1,25 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
```

Comment:
These lines aren't necessary. This module won't be run as a script, and it contains no Unicode.